addons/pos_hr/static/src/app/login_screen/login_screen.xml

```diff
@@ -1,20 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="pos_hr.LoginScreen">
-        <div class="login-overlay fixed-top w-100 h-100 bg-view">
-            <div class="screen-login position-absolute top-0 start-0 bottom-0 end-0 d-flex flex-column py-4 m-auto w-100 rounded bg-view text-center fw-bolder">
-                <div class="login-title fs-2 mb-3 mb-lg-0">Log in to
-                    <span class="text-primary" t-esc="pos.config.name" />
+        <div class="login-overlay o_attendance_background fixed-top w-100 h-100">
+            <div class="screen-login-header d-flex justify-content-between px-4 pt-4 mb-0 w-100">
+                <div class="screen-login-timer d-flex align-items-center">
+                    <div class="timer-hours d-flex align-items-center" t-att-class="{'fs-6': ui.isSmall, 'fs-1': !ui.isSmall}" t-esc="state.hours"> </div>
+                    <div class="timer-date-container flex-column justify-content-center ms-2 d-none d-lg-flex fs-6">
+                        <div class="timer-day d-flex align-items-center" t-esc="state.day"> </div>
+                        <div class="timer-date d-flex align-items-center" t-esc="state.date"> </div>
+                    </div>
                 </div>
-                <div class="login-body d-flex d-flex flex-column flex-sm-row align-items-center justify-content-around px-3 py-4">
-                    <span class="login-element border rounded">
-                        <img class="login-barcode-img img-fluid"
-                             src="/point_of_sale/static/img/barcode.png" />
-                        <div class="login-barcode-text mt-2">Scan your badge</div>
+                <div class="screen-login-logo ms-auto">
+                    <img t-attf-src="/web/image?model=res.company&amp;id={{pos.company.id}}&amp;field=logo" alt="Logo"/>
+                </div>
+            </div>
+            <div class="screen-login position-absolute top-0 start-0 bottom-0 end-0 d-flex flex-column py-4 m-auto w-100 rounded text-center fw-bolder">
+                <div class="login-body d-flex d-flex flex-column flex-sm-row align-items-center justify-content-center gap-2">
+                    <span class="login-element d-flex flex-column justify-content-center align-items-center bg-white mb-4 rounded-4 mobile-scanner" t-on-click="open_mobile_scanner">
```

Comment:
You are using an enterprise method `t-on-click="open_mobile_scanner"` in the community part. This will not work without the enterprise module installed. Second thing is that when I tested, the scanning of employee did not worked. Can you test and see if the useBarcodeReader hook is used and if it is used in a good way